python/lsst/afw/image/imageSlice.cc

```diff
@@ -1,39 +1,37 @@
-/* 
+/*
  * LSST Data Management System
- * Copyright 2008-2016  AURA/LSST.
- * 
+ * Copyright 2008-2017 AURA/LSST.
+ *
  * This product includes software developed by the
  * LSST Project (http://www.lsst.org/).
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
- * 
+ *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
- * 
- * You should have received a copy of the LSST License Statement and 
- * the GNU General Public License along with this program.  If not, 
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
  * see <https://www.lsstcorp.org/LegalNotices/>.
  */
 
-#include <pybind11/pybind11.h>
-//#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include "pybind11/pybind11.h"
 
 #include "lsst/afw/image/ImageSlice.h"
 
 namespace py = pybind11;
 
 using namespace py::literals;
 
-using namespace lsst::afw::image;
+namespace lsst { namespace afw { namespace image { namespace {
```

Comment:
Perhaps too picky but...do we even allow putting all of these on the same line? clang-format will certainly change it.